src/stdatamodels/jwst/datamodels/schemas/specwcs_miri_lrs.schema.yaml

```
@@ -0,0 +1,118 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.1.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/specwcs_miri_lrs.schema"
+title: MIRI LRS Fixed Slit Specwcs Schema
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_pexptype.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_filter.schema
+- $ref: subarray.schema
+- type: object
+  properties:
+    wavetable:
+      title: Wavelengths and x, y locations of wavelengths
+      fits_hdu: WAVETABLE
+      datatype:
+      - name: x_center
+        datatype: float32
+        unit: pixels
+        title: x location of the center of a given wavelength element
+      - name: y_center
+        datatype: float32
+        unit: pixels
+        title: y location of the center of a given wavelength element
+      - name: wavelength
+        datatype: float32
+        unit: microns
+        title: central wavelength of the wavelength element at x_center, y_center
+      - name: x0
+        datatype: float32
+        unit: pixels
+        title: x location of the upper left corner of a given wavelength element
+      - name: y0
+        datatype: float32
+        unit: pixels
+        title: y location of the upper left corner of a given wavelength element
+      - name: x1
+        datatype: float32
+        unit: pixels
+        title: x location of the upper right corner of a given wavelength element
+      - name: y1
+        datatype: float32
+        unit: pixels
+        title: y location of the upper right corner of a given wavelength element
+      - name: x2
+        datatype: float32
+        unit: pixels
+        title: x location of the lower right corner of a given wavelength element
+      - name: y2
+        datatype: float32
+        unit: pixels
+        title: y location of the lower right corner of a given wavelength element
+      - name: x3
+        datatype: float32
+        unit: pixels
+        title: x location of the lower left corner of a given wavelength element 
+      - name: y3
+        datatype: float32
+        unit: pixels
+        title: y location of the lower left corner of a given wavelength element
+- type: object
+  properties:
+    meta:
+      type: object
+      properties:
+        x_ref:
+          type: number
+          title: x coord of ref position of MIRIM_SLIT
+          default: pixels
+          fits_keyword: IMX
+        y_ref:
+          type: number
+          title: y coord of ref position of MIRIM_SLIT
+          default: pixels
+          fits_keyword: IMY
+        x_ref_slitless:
+          type: number
+          title: x coord of ref position of MIRIM_SLITLESS
+          default: pixels
+          fits_keyword: IMXSLTL
+        y_ref_slitless:
+          type: number
+          title: y coord of ref position of MIRIM_SLITLESS
+          default: pixels
```

```suggestion
          default: pixels
```
Is this supposed to be units? This results in a default value of 'pixels' when creating a reference model:
```python
>> m = dm.MiriLRSSpecwcsModel()
>> m.meta.y_ref_slitless
'pixels'
```
There are a few other instances of this in the schema.

---

I missed that these are not in an array datatype block and are instead for properties (datamodel attributes).

`units` isn't an understood schema keyword (nor is `unit`). I believe the convention for keywords is to put the unit before the title in square brackets. Something like:
```yaml
title: "[pixels] y coord of ref position of MIRIM_SLITLESS"
```